### ISSUE TITLE
AV.Promise failed to catch error in resolveFun

### ISSUE THREAD

**Issue by liuyanghejerry** [leancloud/javascript-sdk#98]
Given code like this:

```javascript
       new AV.Promise(function(resolve, reject) {
		return resolve(123);
	})
	.then(function() {
		throw {code: 1};
	})
	.then(function(){}, function(error) {
		console.log('catch them all!');
	})
```

Expected to print `catch them all!` but got uncaught error.

When inspected the source, found `_isPromisesAPlusCompliant = false` in promise.js, which seems to break the behavior expected earlier. 

Wondering why?

**Comment by liuyanghejerry** [comment]
Also, #34 said Promise/A+ is compliant already, but actually not.

**Comment by killme2008** [comment]
We will release a new version today, and it will fix this issue,thanks for your feedback.

**Comment by killme2008** [comment]
We released v0.5.5 yesterday.Thanks.
